crates/swc_ecma_minifier/src/compress/pure/strings.rs

```diff
@@ -23,6 +23,7 @@ where
                 }) && (M::force_str_for_tpl()
                     || (!c.value.contains("\\n") && !c.value.contains("\\r")))
                     && !c.value.contains("\\0")
+                    && !c.value.contains("\\x")
```

Comment:
before this, we store 
```js
`\x21`
```
like \0x21, hit line 25.
now it is \x21 so I add this line